context/app/static/js/components/Detail/Visualization/Visualization.jsx

```diff
@@ -97,20 +107,35 @@ function Visualization(props) {
       </StyledHeader>
       <Paper>
         <ExpandableDiv $isExpanded={isExpanded} $theme={vitessceTheme}>
-          <TopSnackbar
+          <EscSnackbar
             anchorOrigin={{
               vertical: 'top',
               horizontal: 'center',
             }}
-            open={isSnackbarOpen}
+            open={isEscSnackbarOpen}
             autoHideDuration={4000}
-            onClose={() => setIsSnackbarOpen(false)}
+            onClose={() => setIsEscSnackbarOpen(false)}
             message="Press [esc] to exit full window."
           />
+          <WarnSnackbar
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'right',
+            }}
+            open={isWarnSnackbarOpen}
+          >
+            <Alert severity="warning" variant="filled" onClose={() => setIsWarnSnackbarOpen(false)}>
+              {vitessceWarning}
+            </Alert>
+          </WarnSnackbar>
           <Vitessce
             config={vitData[vitessceSelection] || vitData}
             theme={vitessceTheme}
             height={isExpanded ? null : vitessceFixedHeight}
+            onWarn={(warning) => {
+              setVitessceWarning(warning);
```

Comment:
I'm wondering about the case where multiple warnings come in together... perhaps they should be added to an array, and closing the snackbar clears and resets the array?

Reply:
I have added the array, and closing snackbar removes a particular warning from the array so that the next warning can be shown if there is more than one